core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java

```diff
@@ -33,31 +34,24 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
-import javax.persistence.Table;
 import javax.persistence.Transient;
 
-import org.apache.commons.lang.builder.EqualsBuilder;
-import org.apache.commons.lang.builder.HashCodeBuilder;
-
 import com.eaglegenomics.simlims.core.User;
 
 import uk.ac.bbsrc.tgac.miso.core.data.ChangeLog;
 import uk.ac.bbsrc.tgac.miso.core.data.type.KitType;
 import uk.ac.bbsrc.tgac.miso.core.data.type.PlatformType;
 
 /**
- * A KitDescriptor handles information about a consumable element that is generally typed by a name, manufacturer and part number. Kits use
- * KitDescriptors, coupled with a lot number, to represent a real-world manifestation of a consumable kit.
- * 
- * @author Rob Davey
+ * A KitDescriptor handles information about a consumable element that is generally typed by a name, manufacturer and part number.
+ * KitComponents use
+ * KitComponentsDescriptors, which in turn use KitDescriptors to represent a real-world manifestation of a consumable kit.
+ *
+ * @author Rob Davey, Michal Zak
  * @since 0.0.2
  */
 @Entity
-@Table(name = "KitDescriptor")
 public class KitDescriptor implements Serializable {
-
```

Comment:
Since you're deleting Notes from Kit, you might want to consider putting them on KitDescriptor -- the lab might want to be able to attach notes to all specific kits of a certain type (worth at least checking this with them).